src/content/docs/en/pages/guides/edge-sql/create-database.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: How to manage an Edge SQL database
+description: Learn how to create, list, and delete databases from Edge SQL using the Azion REST API.
+meta_tags: edge sql, sql, database, crud
+namespace: docs_sql_database_crud
+permalink: /documentation/products/guides/manage-databases-edge-sql/
+---
+
+import Button from '~/components/Button.astro'
+import Badge from '~/components/Badge.astro'
+
+<Badge variant="accent">
+  Preview
+</Badge>
+
+**Edge SQL** allows you to create databases so you can insert data. This guide covers how to create, list, and delete a database using the [Azion API](https://api.azion.com).
+
+<Button href="/en/documentation/products/store/edge-sql/" text="go to Edge SQL reference" variant="secondary"></Button>
+
+After creating a database, read [How to create an Edge SQL query](/en/documentation/products/guides/create-query-edge-sql/) to populate your database.
+
+---
+
+## Creating a database
+
+Run the following `POST` request in your terminal, replacing `[TOKEN VALUE]` with your [personal token](/en/documentation/products/guides/personal-tokens/) and `name` with the desired database name to create a new database:
+
+```bash
+curl --location 'https://api.azion.com/v4/edge_sql/databases' \
+--header 'Authorization: Token [TOKEN VALUE]' \
+--header 'Content-Type: application/json' \
+--data '{
+   "name": "my-database"
+}'
+```
+
+You should receive the following response:
+
+```json
+{
+  "state": "pending",
+  "data": {
+    "id": 118,
+    "name": "my-database",
+    "client_id": "6832h",
+    "status": "creating",
+    "created_at": "2024-04-18T11:22:59.468536Z",
+    "updated_at": "2024-04-18T11:22:59.468586Z",
+    "deleted_at": null
+  }
+}
+```
+
```

Comment:
Maybe add some information about this action not being instantaneous. When the status changes from pending to created  you will be able to use the created database.